manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
```

Comment:
I thought the same but the thing is that the packages provided by this custom repo are now part of official repositories, the question is if we want to support installation on older RHEL/CentOS releases